circle.yml

```diff
@@ -0,0 +1,63 @@
+version: 2
+jobs:
+
+  xcode_9_swift_3:
+    macos:
+      xcode: "9.0"
+    steps:
+      - checkout
+      - run:
+          command: |
+            ( xcodebuild -scheme Yams test SWIFT_VERSION=3.2 # -sdk macosx -destination arch=x86_64
+              xcodebuild -scheme Yams test SWIFT_VERSION=3.2 OTHER_SWIFT_FLAGS=-DUSE_UTF8
+              xcodebuild -scheme Yams test SWIFT_VERSION=3.2 -sdk iphonesimulator -destination "name=iPhone 8"
+              xcodebuild -scheme Yams test SWIFT_VERSION=3.2 -sdk appletvsimulator -destination "name=Apple TV 4K"
+              xcodebuild -scheme Yams build SWIFT_VERSION=3.2 -sdk watchsimulator -destination "name=Apple Watch Series 3 - 38mm"
+            ) | xcpretty -r junit
+      - store_test_results:
+          path: build/reports/
+
+  xcode_9_swift_4:
+    macos:
+      xcode: "9.0"
+    steps:
+      - checkout
+      - run:
+          command: |
+            ( xcodebuild -scheme Yams test # -sdk macosx -destination arch=x86_64
+              xcodebuild -scheme Yams test OTHER_SWIFT_FLAGS=-DUSE_UTF8
+              xcodebuild -scheme Yams test -sdk iphonesimulator -destination "name=iPhone 8"
+              xcodebuild -scheme Yams test -sdk appletvsimulator -destination "name=Apple TV 4K"
+              xcodebuild -scheme Yams build -sdk watchsimulator -destination "name=Apple Watch Series 3 - 38mm"
+            ) | xcpretty -r junit
+      - store_test_results:
+          path: build/reports/
+
+  spm_swift_4:
+    macos:
+      xcode: "9.0"
+    steps:
+      - checkout
+      - run: swift test
+      - run: swift test -Xswiftc -DUSE_UTF8
+
+  linux_swift_4:
+    docker:
+      - image: norionomura/swiftlint:swift-4.0
```

Comment:
Since `swift-4.0.2-RELEASE` appeared,` swift-4.0` changed to point to Swift 4.0.2. When explicitly specifying Swift 4.0 same as Xcode 9, we need to write `swift-4.0.0`.